openmetadata-ui/src/main/resources/ui/src/components/common/EntityPageInfos/AnnouncementDrawer/AnnouncementDrawer.tsx

```diff
@@ -64,92 +61,96 @@ const AnnouncementDrawer: FC<Props> = ({
     </Space>
   );
 
-  const createThread = async (data: CreateThread) => {
-    try {
-      await postThread(data);
-    } catch (err) {
-      showErrorToast(err as AxiosError);
-    }
-  };
-
-  const deletePostHandler = (
+  const deletePostHandler = async (
     threadId: string,
     postId: string,
     isThread: boolean
-  ) => {
-    deletePost(threadId, postId, isThread);
+  ): Promise<void> => {
+    await deletePost(threadId, postId, isThread);
   };
 
-  const postFeedHandler = (value: string, id: string) => {
+  const postFeedHandler = async (value: string, id: string): Promise<void> => {
     const data = {
       message: value,
       from: currentUser?.name,
     } as Post;
-    postFeedById(id, data).catch((err: AxiosError) => {
-      showErrorToast(err);
-    });
+
+    try {
+      await postFeedById(id, data);
+    } catch (err) {
+      showErrorToast(err as AxiosError);
+    }
   };
 
-  const updateThreadHandler = (
+  const updateThreadHandler = async (
     threadId: string,
     postId: string,
     isThread: boolean,
     data: Operation[]
-  ) => {
+  ): Promise<void> => {
     const callback = () => {
       return;
     };
 
-    updateThreadData(threadId, postId, isThread, data, callback);
+    await updateThreadData(threadId, postId, isThread, data, callback);
```

Comment:
same wrap in try catch

Reply:
@karanh37 `updateThreadData ` function already contains try-catch handler, so no need to wrap this here!